### ISSUE TITLE
Error loading template

### ISSUE THREAD

**Issue by telecom2k3** [Offerel/roundcube_caldavzap#2]
No errors in Apache logs, or in Chrome's console.  I get this error in the Roundcube errors log:

PHP Error: Error loading template for caldavzap.caldavzap in /var/www/roundcube/program/include/rcmail_output_html.php on line 614 (GET /?_task=caldavzap)

Ubuntu/apache-2.4/PHP-7.1/Roundcube-1.3.3

**Comment by Offerel** [comment]
Is the path to your plugin /var/www/roundcube/plugins/caldavzap/ or /var/www/roundcube/plugins/roundcube_caldavzap/ ?

**Comment by telecom2k3** [comment]
Yes, /var/www/roundcube/plugins/caldavzap.  I changed the path in the plugin's root folder, config.inc.php, file and it is working now.  Thank you!

Also note that using any other theme other than Larry breaks this plugin also.  I simply copied the Caldavzap plugin folder in "skins" and renamed it to match the names of other available skins and it fixes the problem.

**Comment by Offerel** [comment]
support for classic skin is now included and should work.
